index.html

```diff
@@ -15,17 +15,190 @@ <h2 class="flex"><a href="#" target="_blank">John Doe</a></h2>
     <img class="hover-button" src="Icon-Menu.png" alt="">
   </nav>
 
-  <section>
+  <section class="bg-image">
     <h1>Christian Contreras</h1>
 
     <h2>Software Developer</h2>
 
-    <p>
+    <p class="description">
       I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
     </p>
   </section>
 
   <hr>
+
+  <Section class="links flex" >
+    <div class="language-links">
+      <h3>Language</h3>
+
+      <p>Javascript</p>
+      <p>Ruby</p>
+      <p>Html</p>
+      <p>Css</p>
+    </div>
+    
+    <div class="frameworks-links">
+      <h3>Frameworks</h3>
+
+      <p>Bootstrap</p>
+      <p>Ruby on Rails</p>
+      <p>RSPec</p>
+      <p>CapyBara</p>
+      <p>Selenium</p>
+    </div>
+    
+    <div class="skills-links">
+      <h3>Skills</h3>
+
+      <p>Codekit</p>
+      <p>Github</p>
+      <p>Codepen</p>
+      <p>Gitlab</p>
+      <p>Terminal</p>
```

Comment:
Replace the p tags with ul tags because the usage of p tag meant to be only for paragraphs